resources/js/components/requests/requestModal.vue

```diff
@@ -0,0 +1,259 @@
+              </g>
+            </g>
+          </svg>
+        </div>
+      </div>
+      <h4>{{ $t('Finding Requests available to you...') }}</h4>
+    </div>
+
+    <pagination
+      ref="listProcess"
+      :single="$t('Process')"
+      :plural="$t('Processes')"
+      :per-page-select-enabled="true"
+      @changePerPage="changePerPage"
+      @vuetable-pagination:change-page="onPageChange"
+    />
+  </b-modal>
+</template>
+
+<script>
+import card from "./card";
+import datatableMixin from "../common/mixins/datatable";
+
+export default {
+  components: {
+    "process-card": card,
+  },
+  mixins: [datatableMixin],
+  props: {
+    permission: Array,
+    url: "",
+    size: "",
+  },
+  data() {
+    return {
+      filter: "",
+      loading: false,
+      error: false,
+      loaded: false,
+      processes: {
+        // Blank
```

Comment:
If it is possible, to prevent reactivity problems, consider to add the used properties of this Object.

Reply:
this is old code, we only move a other file in order to use in web and mobile